StanfordCPPLib/collections/deque.h

```diff
@@ -362,6 +362,13 @@ class Deque {
      * This is used to check for invalid iterators and issue error messages.
      */
     unsigned int version() const;
+    
+    /* Assignment behavior. */
+    Deque& operator= (Deque other) {
```

Comment:
nit: I've tried to standardize the whitespacing of overloaded operators to put a space after the word "operator" and not before the parenthesis.  Also, should "other" be a const reference rather than a value copy?

```
Deque& operator =(const Deque& other) {
```